src/core/ext/transport/binder/transport/binder_transport.cc

```diff
@@ -172,6 +172,20 @@ static void cancel_stream_locked(grpc_binder_transport* gbt,
   GRPC_ERROR_UNREF(error);
 }
 
+static bool ContainsAuthorityAndPath(const grpc_binder::Metadata& metadata) {
+  bool has_authority = false;
+  bool has_path = false;
+  for (const auto& kv : metadata) {
+    if (kv.first == grpc_core::StringViewFromSlice(GRPC_MDSTR_AUTHORITY)) {
+      has_authority = true;
+    }
+    if (kv.first == grpc_core::StringViewFromSlice(GRPC_MDSTR_PATH)) {
+      has_path = true;
+    }
+  }
+  return has_authority && has_path;
+}
+
```

Comment:
There are the relevant changes of this PR. @ctiller can you confirm that this check is indeed required and correct?

Reply:
Path is certainly required... although Authority feels much more like a http thing. Can you link to the code that was asserting?

Reply:
I believe so.